Documentation/MigrationGuide.md

```diff
@@ -0,0 +1,21 @@
+## StencilSwiftKit 2.0 (SwiftGen 5.0) ##
+
+### For template writers:
+
+* We've removed our `join` array filter as Stencil provides it's own version that accepts a parameter. If you were using StencilSwiftKit's version, replace instances of:  
+```{{ myArray|join }}```  
+with:  
+```{{ myArray|join:", " }}```
+* We've refactored our `snakeToCamelCase` to accept arguments, thus removing the need for `snakeToCamelCaseNoPrefix`. If you were using the latter, replace instances of:  
+```{{ myValue|snakeToCamelCaseNoPrefix }}```  
+with:  
+```{{ myValue|snakeToCamelCase:true }}```
+
+### For developers:
```

Comment:
Hehe, I know 😆 I'll add that (without the Swift part, that should be clear enough), clarity is more important than nice titles.